src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/CustomerSettings/Title/Blocks/form.html.twig

```diff
@@ -0,0 +1,50 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% form_theme titleForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+
+{% block title_form %}
+  {{ form_start(titleForm) }}
+    <div class="card">
+      <h3 class="card-header">
+        <i class="material-icons">man</i>
+        {{ 'Social titles'|trans({}, 'Admin.Shopparameters.Feature') }}
```

Comment:
`Social title` or `Title`, we are editing a single entity.

Reply:
Ping @l-delin for confirmation

Reply:
Context : It's the page "Add a new title"

Reply:
Hello! 
We should use "Title" as we only refer to using Mr. Mrs. (see screenshot) and not to other titles [(Lord, Count, Duke...)](https://en.wikipedia.org/wiki/Category:Men%27s_social_titles):
<img width="724" alt="image" src="https://user-images.githubusercontent.com/113511191/221541128-331424f5-580a-4294-b083-09addfb26dba.png">

Following those examples, `Social title` doesn't make a lot of sense here, so we should use `Title`. What do you think, @Hlavtox ?

Reply:
@l-delin I think we should use `Title`.

From a quick google I think that:
- Gender: Male, Female
- Title: Mr., Mrs., Professor, Sir

Reply:
Let's change it to `Title`, then 🎉